modules/planning/tasks/rss/decider_rss_test.cc

```diff
@@ -18,51 +18,59 @@
  * @file
  **/
 
-#include "rss/core/RssCheck.hpp"
-#include "rss/test_support/TestSupport.hpp"
 
-namespace rss {
+#include "ad_rss/core/RssCheck.hpp"
+#include "TestSupport.hpp"
```

Comment:
I still vote strongly to replace TestSupport functionality from production code. use inside the tests is ok for sure. See above on how to handle this.

Reply:
@berndgassmann  I removed the test support from decider_rss and please refer to the patch "rss-open-library-wrapper-remove-test-header"
As for decider_rss_test.cc,  let's keep it first and remove the test header in next release.